Documentation/configuration.md

```diff
@@ -130,6 +130,10 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 + default: none
 + env variable: ETCD_DISCOVERY_PROXY
 
+##### -strict-reconfig-check
++ Reject reconfiguration requests if the requests can cause quorum loss.
```

Comment:
Possible rewording:

"Reject reconfiguration requests that would cause quorum loss."